hw1

i-baumann opened the pull request:
adding hw1 file, updating requirements, extending flake8 character limit

i-baumann committed:
hw1

adding hw1 file, updating requirements, extending flake8 character limit

i-baumann committed:
adding .sh

adding .sh script for requirements install and running hw1-fisher.py

i-baumann committed:
removing label encoding

removing unecessary label encoding of flower class

colinanderson10 reviewed (commented):
Everything works perfectly when running, the only suggestion I have is to add some numpy descriptives in addition to the simpler pandas describe method

i-baumann committed:
adding numpy sum stats

adding numpy summary stats just to be safe, using special VLSJ* (TM) output *very long super janky

dafrenchyman reviewed (approved):
The code would have been better organized if there was a main function. The function that was inline would have been separated out and been cleaner too. Other than that, great work. Clean, easy to follow, etc…Great job.
